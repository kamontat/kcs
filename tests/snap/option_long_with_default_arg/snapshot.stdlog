[DBG] init.options         reuse definitions from cache file '$TMPDIR/kcs/libs-options/options-definitions.txt'
[DBG] hook-init.options    options definition: |-l|:LAND:NV::;|-f|--flower|:FLOWER:NV::;|-s|--sky|:SKY:RS::;|-w|--water|:WATER:OS::;|-r|--rock|:ROCK:OS:paper:;|-e|--ex|--extra|:EXTRA:NV:paper:;|--q|:QUE:NV:paper:;
[DBG] hook-init.options    parsing short options '-r'
[DBG] checker.options      option '-r' not provide any argument
[DBG] export.options       assign 'paper' as default argument to option '-r'
[DBG] export.options       export '_KCS_OPT_ROCK_VALUE'='paper'
