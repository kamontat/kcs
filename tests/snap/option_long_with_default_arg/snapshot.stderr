[ERR] checker.options      option '|-r|--rock|' contained default argument and no custom argument provided
[ERR] exit.base            loading 'func:__kcs_options_hook_init' failed
[ERR] exit.base            loading 'commands:_options' failed ($KCT_PATH_TESTDIR/commands/_options.sh)
